### ISSUE TITLE
Why do I have to put the employeeID the company record?

### ISSUE THREAD

**Issue by maikolachi** [primac/primac-nesq#1]
db.companies.insert(
    {
        "cono" : "001",
        "name" : "vercom",
        "database" : "PRIMAC-001",
        "accountingMethod" : "Cash",
        "measurementSystem" : "US",
        "employeID" : "101",
        "street" : ["13355 Noel Rd", "Suite 1100"]
        "city" : "Dallas",
        "state" : "TX",
        "zip" : "75240",
        "country" : "USA",
        "checksum" : "49012345671"
    }
)

Should it not be the employee_oid? But why is it needed.

**Comment by primac-maquil** [comment]
Yes it should not be there. I have removed it from readme.me file under primac-nesq development branch.
